docs/pages/push-notifications/push-notifications-setup.mdx

```diff
@@ -8,6 +8,8 @@ import { Terminal } from '~/ui/components/Snippet';
 import { Step } from '~/ui/components/Step';
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 
+> Expo Notifications only supports the **Cloud Messaging API (Legacy)** key at this time. This key is deprecated by Firebase. However, it will continue to work until June 20, 2024. We will provide information on migrating to the new v1 key in the future.
```

Comment:
Should we mention we'd anticipate this to happen in Q1 next year?

Reply:
I'm not sure about that. We don't have a specific date set so I'd avoid it but don't have a personal opinion here.